neurons/text/prompting/validators/core/gating.py

```diff
@@ -113,7 +113,8 @@ def forward( self, message: str ) -> 'torch.FloatTensor':
                     Scores for each uids as output by the gating model.
         """
         inputs = self.tokenizer( message, return_tensors="pt" ).to( self.device )
-        hidden_states = self.model( **inputs ).last_hidden_state[0, -1, :]
+        with torch.no_grad():
```

Comment:
Would the gating model not need gradients here in order to be trained?

Reply:
The idea is that we don't have enough data to fully fine-tune the transformer block. Instead, we should only train the last linear layer and freeze the current state of the transformer.